clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/processor/CloudLoggingProcessor.java

```diff
@@ -71,6 +73,14 @@ public AuditLogRequest process(AuditLogRequest auditLogRequest) throws LogProces
                           new TypeReference<Map<String, ?>>() {})))
               .setLogName(getLogNameFromLogType(auditLogRequest.getType()))
               .setLabels(auditLogRequest.getLabelsMap())
+              .setOperation(
+                  Operation.of(
+                      auditLogRequest.getOperation().getId(),
```

Comment:
I saw that `operation` object also has `first` and `last` field

Reply:
Yes, we only need id and producer according to here: https://github.com/abcxyz/lumberjack/blob/563a9dd75e25d3645e6bebecc506d7770679f17c/clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/AuditLoggingServerInterceptor.java#L125

Reply:
Can we add a code comment to explain why we only need these two fields?

Reply:
Yes, will do.

Reply:
Changed to add all fields to cover http client case.